cmr-exchange/ous-plugin/src/cmr/ous/impl/v2_1.clj

```diff
@@ -30,21 +30,24 @@
   [component coll-promise grans-promise {:keys [endpoint token params]}]
   (log/debug "Starting stage 2 ...")
   (let [granules (granule/extract-metadata grans-promise)
-        coll (collection/extract-metadata coll-promise)
-        tag-data (get-in coll [:tags (keyword collection/opendap-regex-tag) :data])
+        gran-headers (granule/extract-header-data grans-promise)
+        coll-body (collection/extract-body-metadata coll-promise)
+        tag-data (get-in coll-body [:tags (keyword collection/opendap-regex-tag) :data])
         granule-links (map granule/extract-granule-links granules)
-        service-ids (collection/extract-service-ids coll)
-        vars (common/apply-bounding-conditions endpoint token coll params)
+        sa-header (get-in gran-headers [:cmr-search-after])
+        hits-header (get-in gran-headers [:cmr-hits])
```

Comment:
Could do `(get gran-headers :cmr-search-after)` and `(get gran-headers :cmr-hits)` instead of using `(get-in` since you're only using a single key.